bin/pycbc_single_template

```diff
@@ -315,6 +317,10 @@ with ctx:
                                     f_lower=template_flow, delta_f=delta_f,
                                     delta_t=gwstrain.delta_t)
             template = tc.multiply_and_add(tp, opt.u_val)
+    
+    kmax = int(opt.high_frequency_cutoff / template.delta_f)
```

Comment:
@MarlinSchaefer You should put this in check that the option was given, since we don't want it to be required. 

i.e. 

```
if opt.high_frequency_foff:
      ....
```

Reply:
Added the check